src/containerapp/azext_containerapp/containerapp_sessionpool_decorator.py

```diff
@@ -415,17 +416,32 @@ def set_up_container(self, customer_container_template):
 
     def set_up_registry_auth_configuration(self, secrets_def, customer_container_template):
         if self.get_argument_registry_server() is not None:
-            safe_set(customer_container_template, "registryCredentials", "registryServer", value=self.get_argument_registry_server())
+            safe_set(customer_container_template, "registryCredentials", "server", value=self.get_argument_registry_server())
         if self.get_argument_registry_user() is not None:
             safe_set(customer_container_template, "registryCredentials", "username", value=self.get_argument_registry_user())
         if secrets_def is None:
             secrets_def = []
         if self.get_argument_registry_pass() is not None:
+            original_secrets = self.existing_pool_def["properties"]["secrets"]
+            original_secrets_names = []
+            for secret in original_secrets:
+                original_secrets_names.append(secret["name"])
+            logger.warning(original_secrets)
```

Comment:
We should remove this `logger.warning(original_secrets)`, right?

Reply:
Updated.